docs/modeling/relational-table.rst

```diff
@@ -0,0 +1,53 @@
+Relational Databases: Table Mapping
+===================================
+
+Table mapping identifies which table data should be queried from and saved to in the database.
+
+.. note::
+    The configuration in this section is applicable to relational databases in general. The extension methods shown here will become available when you install a relational database provider (due to the shared *EntityFramework.Relational* package).
+
+.. contents:: In this article:
+    :depth: 3
+
+Conventions
+-----------
+
+By convention, each entity will be setup to map to a table with the same name as the entity.
```

Comment:
May be worth mentioning that SQLite's table names are case-insensitive, so adding mappings for casing-differences is unnecessary.